scripts/prepack.ts

```diff
@@ -81,4 +88,25 @@ try {
   process.exit(1);
 }
 
-console.log(`\nSuccessfully finished prepack commands for ${pkgJson.name}\n`);
+async function runPackagePrepack(packagePrepackPath: string): Promise<void> {
+  const { prepack } = await import(packagePrepackPath);
+  if (prepack && typeof prepack === 'function') {
+    prepack(buildDir);
```

Comment:
If rather have it so that the package specific prepack:

1) returns a Boolean that represent if it was successful or not (essentially result type)
2) throws an error which we can try catch and handle

not the biggest fan of the child prepack calling `process.exit(1);`. What do you think?